deps/wazuh_testing/wazuh_testing/end_to_end/monitoring.py

```diff
@@ -26,7 +26,8 @@
 from wazuh_testing.tools.system import HostManager
 
 
-def monitoring_events_multihost(host_manager: HostManager, monitoring_data: Dict, ignore_error: bool = False) -> Dict:
+def monitoring_events_multihost(host_manager: HostManager, monitoring_data: Dict, ignore_error: bool = True,
+                                scan_interval: int = 5) -> Dict:
```

Comment:
Should we ignore errors by default?

Reply:
`scan_interval` could use `T_5`. In addition, it is missing in the function documentation.

Reply:
**Regarding the ignore_error**

The current naming suggests the wrong functionality. It doesn't actually ignore errors; rather, when enabled, it prevents a TimeoutError from being triggered if some regex patterns aren't found. This functionality is particularly useful in the multithreading environment of E2E tests. Sometimes, we need to monitor various regex patterns on different hosts, and halting monitoring with an exception would hinder this process.
I suggest renaming this variable



**Regarding the use of `T_5`**

The use of the `T_5` is not recommended in this case due to:
- The import of IT modules could lead into unexpected errors. Check https://github.com/wazuh/wazuh-jenkins/issues/6234, for more information
- `T_5` is not a meaningful variable. I suggest using a variable like `DEFAULT_SCAN_INTERVAL`

Reply:
Done d0068357a52f1d59580f3777ab4e35db30321570